.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     runs-on: "ubuntu-latest"
     strategy:
       matrix:
-        python-version: ["3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.12"]
```

Comment:
Is there any value in running against 3.10 and 3.11 here as well or are you assuming that there aren't major changes between those versions that affect the SDK so it isn't necessary to run the suite against them?

Reply:
The benefit seems really small compared to the downside of extra jobs making CI slower